frontend/src/component/admin/menu/AdminMenu.tsx

```diff
@@ -11,6 +10,8 @@ function AdminMenu() {
     const { isBilling } = useInstanceStatus();
     const { flags } = uiConfig;
 
+    const activeTab = pathname.split('/')[2];
```

Comment:
Simplifies the logic, grabs routes with 1 level deep sub-routes like `/network/traffic`.